build.gradle.kts

```diff
@@ -30,8 +30,10 @@ kotlin {
         }
     }
 
-    linuxArm64()
     linuxArm32Hfp()
+    linuxMips32()
+    linuxMipsel32()
+    linuxArm64()
```

Comment:
Now Arm64 comes after Arm32 :+1: In total I'm not sure this new list is an improvement when it comes to alphabetical ordering, though :wink:

Reply:
Tell me what you prefer